packages/react-components/react-button/library/src/components/SplitButton/useSplitButtonStyles.styles.ts

```diff
@@ -195,17 +205,20 @@ export const useSplitButtonStyles_unstable = (state: SplitButtonState): SplitBut
   const focusStyles = useFocusStyles();
   const iconOnlyStyles = useIconOnlyStyles();
 
-  const { appearance, disabled, disabledFocusable } = state;
+  const { appearance, disabled, disabledFocusable, size } = state;
 
-  const noIconClass = state.primaryActionButton
+  const iconOnlyClassName = state.primaryActionButton
     ? !state.primaryActionButton.children && appearance && iconOnlyStyles[appearance]
     : undefined;
 
+  const forceLargeChevron = size === 'medium' && (appearance === 'transparent' || appearance === 'subtle');
```

Comment:
_nit:_ Name it as a conditional

```suggestion
  const shouldForceLargeChevron = size === 'medium' && (appearance === 'transparent' || appearance === 'subtle');
```